ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/ovn/provision/cucushift-installer-rehearse-gcp-upi-ovn-provision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-upi-ovn-provision
+  steps:
+  - ref: ipi-install-rbac
+  - chain: upi-conf-gcp
```

Comment:
Since the chain name contains string `ovn` so I think we should add `ref: ovn-conf` here. And I believe the workflow `cucushift-installer-rehearse-gcp-upi-ovn` is only used for 4.11 and before release.
cc @zhaozhanqi

Reply:
+1

Reply:
Fixed it!